prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div><a href="file:///C:/Users/Paulina%20Hervas/repos/prep-90921-code-solutions/prep-html-links/index.html">Click here to go to the home page</a></div>
```

Comment:
This is an absolute file path, we want to avoid using absolute file paths in our code because it is specific to your computer and won't work on other systems. Here we want to use a relative path. So it would be `href="../index.html"`